### DISCUSSION TITLE
Filetype detection

### DISCUSSION THREAD

**Discussion by skoch13** [jellydn/hurl.nvim#55] [Q&A] [answered]
What is the proper filetype detection should be made on a users side? If I force 'hurl' for 'hurl-nvim' results buffer it looks off to me 
![Screenshot 2024-01-25 at 12 58 08](https://github.com/jellydn/hurl.nvim/assets/29177689/afff0d6e-45ef-4689-a2bf-beb5b375b9c4)

**Comment by jellydn** [comment]
The `hurl-nvim` is for the split view which is just a way for `edgy` to detect the buffer and arrange the layout position. Technically, there are 2 sections: top is `bash` which contains header and bottom is `automatically` detect by the`hurl response` (json, html or text).

**Reply by skoch13** [reply]
Thank you for the answer. So, what is the suggested way of configuring file type detection?

**Reply by jellydn** [reply, marked as answer]
I've just pushed the change. Could you give it a try and let me know if there are any issues? I often use the `nightly` version so I am not aware of the issue with filetype detection.

**Reply by skoch13** [reply]
Thank you. It works properly on nightly
